tests/unit/common/test_table.py

```diff
@@ -946,8 +947,8 @@ def test_init(self):
         self.assertIsNone(table.col6)
         self.assertIsNone(table.col8)
         self.assertIsNone(table.col8_index)
-        self.assertIsNone(table.col10)
-        self.assertIsNone(table.col10_index)
+        self.assertIsNone(table.col11)
+        self.assertIsNone(table.col11_index)
```

Comment:
I'm confused -- columns 2, 4, 6, and 8 are optional so they are not generated by the class when there is no data. But columns 10 and 11 are required, so they should not be None, right?

Reply:
10 and 11 are not required. The description is misleading.

Reply:
Oops, you're right